### PR TITLE
Set PV owner reference

### PR DESCRIPTION
**Pull request by h0tbird**
When a `PV` is created it is set to be dependent of the `Node` backing it.
When the `Node` is deleted the `PV` is also deleted by the k8s garbage collector.

### CODE REVIEW

**Comment by msau42** [comment]
/ok-to-test

We should consider if there's any use cases for leaving the PVs around, and would any users be depending on this current behavior.  If there are such use cases, then this may need to enabled as a new option to not break backwards compatibility.

cc @cofyc @gnufied

**Comment by gnufied** [comment]
I can't think of a reason of not doing this. so +1 from me.

**Comment by cofyc** [comment]
add an option for this?

**Comment by cofyc** [comment]
LGTM

@msau42 What do you think?

File: pkg/common/common.go [hunk 1]
```diff
@@ -199,6 +203,8 @@ type ProvisionerConfiguration struct {
 	// LabelsForPV could be used to specify additional labels that will be
 	// added to PVs created by static provisioner.
 	LabelsForPV map[string]string `json:"labelsForPV" yaml:"labelsForPV"`
+	// SetPVOwnerRef indicates if PVs should be dependents of the owner Node, default to false
+	SetPVOwnerRef bool `json:"setPVOwnerRef" yaml:"setPVOwnerRef"`
```
Comment (msau42) [inline pkg/common/common.go]: Can we add this option to the helm chart (default off) too?

Comment (h0tbird) [reply pkg/common/common.go]: Sure, done here: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/123/commits/74869f0c8ecc273c543618c1d37892aebb1dfbd3

**Comment by h0tbird** [comment]
The test is failing with the error:
```
The command '/bin/sh -c apt-get update     && apt-get upgrade -y     && clean-install     util-linux     e2fsprogs     bash' returned a non-zero code: 1
```
Not much I can do about it, I will retry later.

**Comment by cofyc** [comment]
Thanks! LGTM. Can you squash the commits a bit?

**Comment by cofyc** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**